packages/slate-sections-plugin/index.js

```diff
@@ -13,9 +14,11 @@ module.exports = class sectionsPlugin {
   }
 
   apply(compiler) {
-    compiler.hooks.emit.tapPromise(
-      'Slate Sections Plugin',
-      this.addLocales.bind(this),
+    compiler.hooks.emit.tapPromise(PLUGIN_NAME, this.addLocales.bind(this));
+
+    compiler.hooks.afterEmit.tapPromise(
```

Comment:
This doesn't need to be async, so just `compiler.hooks.emit.tap()` will work

Reply:
I just added it to the other hook we tap, as it can be done there.